### ISSUE TITLE
Microsoft installer package does not include JDBC Driver for SQL Server

### ISSUE THREAD

**Issue by CosmoRied** [traccar/traccar#4721]
I might be wrong, but the windows installer does not include the required library in the default installer to use Windows Authentication to connect to MSSQL server. It might be because the issue is only noticed recently:

https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-ver15

When I tried to connect using integratedSecurity=true in the connection string, it said that it was valid, but that the java library needs to be downloaded and configured. This needs to be included in the next version please?
```
2021-08-10 12:42:21 ERROR: Main method error - no mssql-jdbc_auth-9.2.1.x64 in java.library.path: C:\Program Files\Java\jdk-16.0.2\bin
```
I got the latest JDK, the latest version of the installer & the latest version of MSSQL studio.

**Comment by tananaev** [comment]
I don't know if we want to include this. What value does it provide?

You should be able to add the library to the classpath if you really need it, so it's not a blocker.

**Comment by CosmoRied** [comment]
> What value does it provide?

Saves a user having to search the internet looking for answers.

> You should be able to add the library to the classpath if you really need it, 

True, but what if someone else needs it?

> so it's not a blocker.

Depends on how you look at yesterdays activities. I certainly had to do some leg work to get the package installed and working. Most installers take care of you.

**Comment by CosmoRied** [comment]
I used to add users to a MSSQL database by hand, choosing passwords for them, and relaying those instructions over the phone. I then realised that I could just tick the box labeled "Windows Authentication" and let them use their already configured user accounts to log into the system. I'd recommend the later system as its clearly more secure. 

> Just for context, you are the first person in ~10 years of the project existence who asked about it.

Azure would probably prefer this method if you're going to host it on cloud infrastructure securely. I'm sure its related to the cost in some way of a MSSQL licence, and or Windows server. Some company's DO use it, and DO have licences and care extremely about security, so I guess its a given then that you'd have to include it in this circumstance... I tend to lean towards the linux option, as I can configure it, but I have a windows box too, so I was curious on if I could make it work with my MSSQL databases... I couldn't so I had to install the latest version of MSSQL that by default makes you change the password of a user the first time you log in. Screw that, why NOT use the windows authentication method?
